contrib/opencensus-ext-azure/opencensus/ext/azure/common/transport.py

```diff
@@ -51,8 +53,11 @@ def _transmit(self, envelopes):
                 },
                 timeout=self.options.timeout,
             )
+        except requests.Timeout:
+            logger.warning('Request time out. Ingestion service may be backed up. Retrying.')
+            return self.options.minimum_retry_interval
         except Exception as ex:  # TODO: consider RequestException
-            logger.warning('Transient client side error %s.', ex)
+            logger.warning('Transient client side error %s. Retrying.', ex)
```

Comment:
"Retrying" text would be displayed after the exception, that could be a pretty long string so most likely forever ignored by people looking at the logs

Reply:
Changed so "retrying" appears first.